com.unity.uiwidgets/Editor/UIWidgetsEditorPanel.cs

```diff
@@ -28,6 +28,7 @@ public class UIWidgetsEditorPanel : EditorWindow, IUIWidgetsWindow {
         }
 
         void Update() {
+            
```

Comment:
empty line?

Reply:
I will delete it. This might come from auth reformat code.